### PR TITLE
Fix NewsblogConfig registration with aldryn-reversions. simple test case

### PR DESCRIPTION
**Pull request by cyberstar**
Newsblog config was not registered with aldryn-reversions correctly, which was leading to issues while restoring article, and it was impossible to recover deleted newsblog config.

### CODE REVIEW

File: aldryn_newsblog/admin.py [hunk 1]
```diff
@@ -153,6 +153,7 @@ def add_view(self, request, *args, **kwargs):
 
 class NewsBlogConfigAdmin(
     AllTranslationsMixin,
+    VersionedPlaceholderAdminMixin,
     PlaceholderAdminMixin,
```
Comment (mkoistinen) [inline aldryn_newsblog/admin.py]: `PlaceholderAdminMixin` is already part of `VersionedPlaceholderAdminMixin`, so, we don't need both.

File: aldryn_newsblog/tests/test_reversion.py [hunk 2]
```diff
@@ -173,3 +175,31 @@ def test_edit_plugin_directly(self):
         response = self.client.get(article.get_absolute_url())
         self.assertContains(response, content1)
         self.assertNotContains(response, content2)
+
+    def test_blog_config_recovery_accessible(self):
+        with transaction.atomic():
```
Comment (mkoistinen) [inline aldryn_newsblog/tests/test_reversion.py]: Does this test OK across SQLite, MySQL and Postgres?

Comment (cyberstar) [inline aldryn_newsblog/tests/test_reversion.py]: Couldn't find any reasons why it shouldn't. Seems that there is only one limitation with ``MySQL`` with not ``InnoDB``, but then transaction stuff is a noop, so shouldn't produce any issues.
Anyway this seems as a standard way of creating revision which ``django-reversion`` seems to use.

Comment (mkoistinen) [inline aldryn_newsblog/tests/test_reversion.py]: :+1:

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**